test/testdata/infer/private_methods.rb.flatten-tree.exp

```diff
@@ -0,0 +1,111 @@
+begin
```

Comment:
flatten-tree is the pass that happens after namer, resolver, and class-flatten (in that order). rewriter comes before all three of those passes.

Would you mind replacing this with a `rewrite-tree` expectation file? That will make it easier to see what the changes from this PR are (because once namer reads private visibilities, it doesn't matter how passes after namer further transform and mangle the tree)